api/lib/application/certification-centers/certification-center-controller.js

```diff
@@ -139,4 +139,25 @@ module.exports = {
     });
     return h.response().code(204);
   },
+
+  async sendInvitationForAdmin(request, h) {
+    const certificationCenterId = request.params.certificationCenterId;
+
+    const invitationInformation = await certificationCenterInvitationSerializer.deserializeForAdmin(request.payload);
+
+    const { invitation, created } = await usecases.createOrUpdateCertificationCenterInvitationForAdmin({
+      email: invitationInformation.email,
+      locale: invitationInformation.lang,
+      certificationCenterId,
+    });
+
+    const serializedData = certificationCenterInvitationSerializer.serializeForAdmin(invitation);
+    let response;
+    if (created) {
+      response = h.response(serializedData).created();
+    } else {
+      response = h.response(serializedData).code(200);
```

Comment:
Ce n'est pas à modifier, mais Hapi a par défaut 200 comme code HTTP si tu fournis un payload de retour sans définir de code.

```suggestion
      response = h.response(serializedData);
```

Cela éviterait un appel à une méthode inutilement.

Reply:
Je suis bien d'accord avec la compréhension de ce que fait Hapi, et au début je n'avais pas mis `.code(200)`. Seulement le test `tests/unit/application/certification-centers/certification-center-controller_test.js` plante si on ne renvoie pas explicitement `200`. J'aimerais vraiment ne pas ajouter `.code(200)` mais il faut alors que je modifie `hFake` de `api/tests/test-helper.js`, je vais investiguer si je peux.

Reply:
Remarque : La création d'une ressource c'est le code 201 à renvoyer